app/controllers/communicarts_controller.rb

```diff
@@ -52,6 +55,9 @@ def approval_response
 private
 
   def validate_access
+    request_from_email = params['email_delivery'].presence || true
+    return unless request_from_email == true
```

Comment:
This logic is kind of confusing. If `params['email_delivery']` is not truthy, we'll get `true` (due of the second half of the or); we'll then return. If `params['email_delivery']` *is* truthy, we won't return. It doesn't look like it's ever truthy, though?